README.adoc

```diff
@@ -261,6 +261,60 @@ Add a link for the tika container in the above command line:
 
     $ docker run --hostname HOSTNAME -p "25:25" -p 80:80 -p "110:110" -p "143:143" -p "465:465" -p "587:587" -p "993:993" --link cassandra:cassandra --link elasticsearch:elasticsearch --link tika:tika --name james_run -t james_run
 
+=== Run James with Java 8 + Guice + Cassandra + ElasticSearch + RabbitMQ
+
+
+==== Requirements
+Built artifacts should be in ./dockerfiles/run/guice/cassandra/destination folder for cassandra.
```

Comment:
Missing `-rabbit` in path (and may be `-rabbitmq` depending on previous comment on product name).